R/helpers-for-tests.R

```diff
@@ -1,4 +1,4 @@
-#' Skip test if not on Metworx or Drone
+#' Skip test if not on Metworx or CI
```

Comment:
Hmm, this description seems incorrect to me.  This helper skips if `PMF_SKIP_VDIFFR` is set to true, but the description is "Skip test if not on Metworx or CI".  That's misleading because that environment variable isn't inherently tied to either Metworx or the CI.  And that's demonstrated by the fact that, before this PR, that variable was set in DRONE and this skip was triggered, while the variable is not set in these GitHub Actions and the skip is _not_ triggered.

How about changing this to "Skip test if \`PMF_SKIP_VDIFFR\` is set to true"?

Also, given the description of `skip_vdiffr` says "[t]his is useful because `vdiffr::expect_doppleganger()` is very fragile and tends to fail with false positives in other environments (like CI and `R CMD CHECK`)", should GitHub Actions set this?  Or, given the actions are passing without the skip, should we wait and see how fragile these end up being?

Side remark: the commit title (37eb86b) for this change is "tests helpers: prefer CI env var to drone", but this doesn't look at the CI environment variable.

Reply:
You know I will say I expected the oldest ci action to fail without this variable set. `vdiffr` differences can cause mismatches in svgs, so I find it odd that oldest passed in both [b57855d](https://github.com/metrumresearchgroup/pmforest/pull/37/commits/b57855db30504200d7110d45bf63cd50b61168d4) and [b20ce83](https://github.com/metrumresearchgroup/pmforest/pull/37/commits/b20ce83cc8b67b491a6ac6dc096253074bd567b4).

I'll adjust the documentation, but am leaning towards:

> given the actions are passing without the skip, should we wait and see how fragile these end up being?